ibm/operator_collection_sdk/playbooks/create_operator.yml

```diff
@@ -104,15 +183,15 @@
       delay: 5
 
     - name: Create SSH Credential Secret
-      ansible.builtin.command: zoscb-encrypt credential-secret -n "{{ credential_name }}" -u "{{ username }}" -s "{{ ssh_key }}"
-      when: encrypt_cli_available is true and username != "" and ssh_key != "" and passphrase == ""
+      ansible.builtin.command: zoscb-encrypt credential-secret -n {{ credential_name }} -u {{ username }} -s {{ ssh_key }}
+      when: encrypt_cli_available is true and username != None and  ssh_key != None and passphrase == None
 
     - name: Create SSH Credential Secret with passphrase
       ansible.builtin.shell: |
-        export PASSPHRASE="{{ passphrase }}"
```

Comment:
These quotes should be added back, as this caused a bug where passphrases with spaces were being split.